styles.css

```diff
@@ -84,3 +85,24 @@ html {
   font-size: 80%;
   white-space: normal;
 }
+
+.html-dark {
+  background: linear-gradient(#000000, #134d71);
+  color: #f7e4bf;
+}
+
+.box-dark {
+  background: #134d71 !important;
```

Comment:
not entirely sure why certain lines required "!important" to be implemented, while others did not

Reply:
has to do with CSS prioritizing -- but I don't totally get what's getting priority by default